globus_cli/config_command/edit.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import print_function
+import click
+import os
+import subprocess
+
+from globus_cli.helpers import common_options
+
+
+@click.command('edit', help='Edit your Globus Config file')
+@common_options(no_format_option=True)
+def edit_command():
+    """
+    Executor for `globus config edit`
+    """
+    editor = os.environ.get('EDITOR', 'nano')
+    subprocess.call([editor, os.path.expanduser('~/.globus.cfg')])
```

Comment:
You can just have [Ubuntu userland on Windows 10 kernel now.](http://www.zdnet.com/article/microsoft-and-canonical-partner-to-bring-ubuntu-to-windows-10/)